src/pages/wallet/wallet.connect/modal/error.js

```diff
@@ -19,14 +19,14 @@ const styles = StyleSheet.create({
   title: {
     color: color.black,
     fontSize: 17,
-    fontFamily: 'Avenir',
+    fontFamily: 'Avenir-Heavy',
```

Comment:
Define a constant instead of duplicating this literal 4 times.

SonarCloud Code Analysis 提示了这个问题，我们需要把所有字体常量放到
font.family.js里，原来的font.ts应该没用可以删除。

使用时用FontFamily.AvenirHeavy引用到字符串常量。

Reply:
搜索工程里的FontFamily，都替换了。